.vscode/launch.json

```diff
@@ -16,6 +16,62 @@
                 }
             ]
         },
+        {
+            "type": "probe-rs-debug",
+            "request": "launch",
+            "name": "Probe-rs Debug Camera",
+            "chip": "ATSAME51J20A",
+            "coreConfigs": [
+                {
+                    "programBinary": "${workspaceFolder}/target/thumbv7em-none-eabihf/debug/camera",
+                }
+            ]
+        },
+        {
+            "type": "probe-rs-debug",
+            "request": "launch",
+            "name": "Probe-rs Debug Power",
+            "chip": "ATSAME51J20A",
+            "coreConfigs": [
+                {
+                    "programBinary": "${workspaceFolder}/target/thumbv7em-none-eabihf/debug/power",
+                }
+            ]
+        },
+        {
+            // "preLaunchTask": "Build Release Communication",
+            "type": "probe-rs-debug",
+            "request": "launch",
+            "name": "Probe-rs Debug Recovery (Release)",
```

Comment:
Kinda confusing since we can either have a debug or release build. I'd remove the Debug part of this and for any other release configs. Also, for boards like the sensor v2 board we need to use the STM32H733VGTx chip. You may notice that the sensor v2 board does not exist in this branch, so I'd like you to practice updating your branch to keep your changes and merge master in so you can have the newest changes available.